### PR TITLE
Update installation and standalone agent content

### PR DESCRIPTION
**Pull request by dedemorton**
Updates observability docs to support changes in https://github.com/elastic/beats/pull/22290.

The build here will fail until the beats PR is merged.

### CODE REVIEW

**Comment by dedemorton** [comment]
@elasticmachine run elasticsearch-ci/docs

File: docs/en/ingest-management/faq.asciidoc [hunk 1]
```diff
@@ -15,7 +15,11 @@ Also read <<ingest-management-troubleshooting>>.
 == Why doesn't my enrolled agent show up in the {fleet} app?
 
 If {agent} was successfully enrolled, but doesn't show up in the *Agents* list,
-it might not be started. You need to <<run-elastic-agent,start {agent}>>.
+it might not be started. Make sure the `elastic-agent` process is running on
+the host. If it's not running, use the <<elastic-agent-run-command,`run`>>
+command to start it.
+
+//REVIEWERS: IS this correct? Or would you use the restart command?
```
Comment (dedemorton) [inline docs/en/ingest-management/faq.asciidoc:22]:
```suggestion
//TODO: Verify that this is correct. Or would you use the restart command?
```

File: docs/en/ingest-management/faq.asciidoc [hunk 2]
```diff
@@ -92,18 +96,24 @@ run {agent}.
 [[i-rebooted-my-host]]
 == How do I restart {agent} after rebooting my host?
 
-On Windows, if you used PowerShell to install {agent} as a service, the agent
-should still be running after rebooting the host.
+//REVIEWERS: I wasn't sure how to test this section because I can't seem to kill
+//elastic-agent. If I kill the process, it just starts up again. Is that
+//expected behavior? Are the details here correct?
```
Comment (dedemorton) [inline docs/en/ingest-management/faq.asciidoc:101]:
```suggestion
//TODO: Verify that this works (it didn't work for me on macOS)
```

**Review by bmorelli25 — APPROVED** [review]

**Comment by dedemorton** [comment]
Note that I plan to address testing/validation changes in a follow-up PR.

**Pull request merged**